File: 03- Subprogramas y recursividad/subprogramas/java/OperacionesAritmeticas.java
public class OperacionesAritmeticas {
    public static long factorial(int n) {
        long factorial = 1;

        for (int i = 1; i <= n; i++) {
            factorial *= i;
        }

        return factorial;
    }

    public static double calcularSerie(int n) {
        double serie = 0;

        int i = 1;
        while (i <= n) {
            serie += i / factorial(i);

            i++;
        }

        return serie;
    }
}

File: 03- Subprogramas y recursividad/subprogramas/python/paquete/operaciones_aritmeticas.py
def factorial(n):
    factorial = 1

    i = 1
    while i <= n:
        factorial *= i

        i += 1

    return factorial

def calcular_serie(n):
    serie = 0

    i = 1
    while i <= n:
        serie += i / factorial(i)

        i += 1

    return serie
